public/app/core/components/AppChrome/MegaMenu/MegaMenuItem.tsx

```diff
@@ -162,6 +162,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
   listItem: css({
     flex: 1,
     maxWidth: '100%',
+    alignItems: 'start',
```

Comment:
curious - what do we need this for? 🤔

Reply:
@ashharrison90 no idea, you added it to the wrapping Stack that I removed

Reply:
removed it